### PR TITLE
Feat: helm chart repo on gitlab pages

### PR DESCRIPTION
**Pull request by adriangonciarz**
### Description
This pull request enables workflow action to automatically build and Helm Charts to Gitlab Pages. Publishing happens when version in the Chart.yaml is updated in the `trunk` branch

### Motivation and Context
[Related Issue](https://github.com/SeleniumHQ/docker-selenium/issues/1563)

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Release Charts
```
Comment (diemol) [inline .github/workflows/release.yml]: Can you please rename this file to `helm-chart-release.yml` please?

**Review by diemol — CHANGES_REQUESTED** [review]
Can you please sign the CLA?

File: .github/workflows/release.yml (hunk 1 above)
Comment (adriangonciarz) [reply .github/workflows/release.yml]: @diemol done

**Review by diemol — APPROVED** [review]
Thank you, @adriangonciarz!

**Pull request merged**